README.md

```diff
@@ -15,19 +15,31 @@ For that reason, we advocate for the usage of a Code of Conduct. We've also lear
 ## Contributing
 > code commits !== the only means to contributions.
 
-The Community Committee is tasked with further building out the Node.js Community. If you're reading this, you're already a part of that community.
+The Community Committee is tasked with further building out the Node.js Community.
+If you're reading this, you're already a part of that community and we'd love to
+have your help!
 
-As a part of the Node.js Community, we'd love to have your help! To **get started** with contributing, you can check out the Issues in this repository labeled "Good First Issue" to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue!
+Before you get started, here's a broad outline of the Community Committee's governance structure:
 
-As a part of the Node.js Community, we'd love to have your help! Before you get started, here's a broad outline of the Community Committee's governance structure:
-
-- Community Commitee (meta-level concerns, cross-cutting with other teams)
+- Community Committee (meta-level concerns, cross-cutting with other teams)
   - Initiatives (focused on specific tasks, independent from the Community Committee). For example, the Website Redesign Initiative, which is focused on a complete redesign of the https://nodejs.org website
   - Working Groups (like initiatives, but more autonomous and broad in scope)
 
-As seen here, most of the community work that immediately affects the project is done within the numerous __initiatives__. We recommend checking the list of initiatives below and getting involved with one that you find interesting! If nothing suits your fancy and you have concrete ideas, just open an issue here! We can help to point you in the right direction.
+As seen here, most of the community work that immediately affects the project is
+done within the numerous **initiatives**. We recommend checking the list of initiatives
+below and getting involved with one that you find interesting! If nothing suits
+your fancy and you have concrete ideas, open an issue here! We can help to point
+you in the right direction.
+
+To **get started** with contributing, you should read the [Contributing Guidelines](./CONTRIBUTING.md)
+document. This document details the roles you can take on. It also includes a guide
+to contributing and links to `good first issue`s where we're looking for help.
+
+If you're interested in participating in the Community Committee directly, you should
+create an issue asking to be an Observer in our next Community Committee meeting.
```

Comment:
Still the same comment about observership.
To be a bit more explicit, I don't think letting that particular way in would be consistent with the new organization. Firstly, it would IMO remain the "default" choice to becoming a CommComm Member, and secondly, it would create some like "second rank Members". We'd risk having members who have "deserved" the seat by actively contributing, becoming a Collaborator, and then be elevated, and Members who would have bypassed this process altogether.

The other arguments for dropping observership have already been discussed in various meetings anyway (symmetrical process with the TSC and encouraging participating in Initiatives, for instance).